### PR TITLE
More Survival Tools Content Mainlining

### PR DESCRIPTION
**Pull request by chaosvolt**
See https://github.com/CleverRaven/Cataclysm-DDA/issues/12430 for a
discussion of the idea. Will outline here what I've decided so far.

Materials:
- Copper material, with corresponding fixes to existing copper items and
  recipes.

Item Tweaks:
- As above, tweaks to existing items that should use copper.
- The boost to sling damage.
- Makeshift macuahuitl has been tweaked to use stone.

Item Additions:
- Boxer briefs.
- Work gloves.
- Wool cloak, loincloth, hand wraps, and foot wraps.
- Garlic.

Recipe tweaks/additions:
- Italian seasoning, because garlic.
- Tweaks to stone hammer, stone shovel, stone axe, and stone adze
  allowing the use of withered plants for binding (like with stone knife).
- Leather pouch allowing the use of fur.
- Option to make sling from woven strings.

In addition, some things done to compliment the content movements:
- About 2 or 3 professions have been given boxer briefs in place of
  boxers.
- Garlic has been expanded into a bulb item and a cloves item for
  planting. Both are usable for Italian seasoning.
- Recipe requirements and outputs of Italian season have been doubled,
  to make the garlic required be 1 bulb/6 cloves instead of 3 cloves. The
  recipe is also no longer autolearned.
- Work gloves recipe is no longer autolearned.

### CODE REVIEW

File: data/json/item_groups.json [hunk 1]
```diff
@@ -3516,6 +3523,7 @@
       ["sleeveless_duster_leather", 2],
       ["sleeveless_duster_fur", 1],
       ["cloak", 5],
+      ["cloak_wool", 5]
```
Comment (Rivet-the-Zombie) [inline data/json/item_groups.json]: Missing a separator here.

**Comment by chaosvolt** [comment]
Oh derp. =w=

**Comment by chaosvolt** [comment]
Now hoping I didn't mess up any more of it.

**Comment by chaosvolt** [comment]
Okay, that would be a sign for me to copy-paste this shit and do a load test BEFORE commiting, next time.

File: data/json/item_groups.json [hunk 2]
```diff
@@ -9057,6 +9071,7 @@
       ["gauntlets_bone", 12],
       ["gloves_fur", 4],
       ["gloves_leather", 22],
+      ["gloves_work", 22]
```
Comment (Rivet-the-Zombie) [inline data/json/item_groups.json]: Another missing separator.

**Comment by Rivet-the-Zombie** [comment]
>Okay, that would be a sign for me to copy-paste this shit and do a load test BEFORE commiting, next time.

Good thought.

**Comment by chaosvolt** [comment]
Okay, cooking up some copy-pasta to test it before I fix that next one, to catch anything else.

Also, addiction type not found...huh. O.o

**Comment by chaosvolt** [comment]
Chestwrap? That's a thing now? owo

**Comment by chaosvolt** [comment]
...fun. :V

**Comment by chaosvolt** [comment]
Most recent compiled experimental apparently derps in other ways when I apply copy-pasting, so still unable to thoroughly copy-paste test this. Hoping I found the last of my fuckups.

**Comment by chaosvolt** [comment]
Flase. Ahahah. Fuck me.

**Comment by Rivet-the-Zombie** [comment]
>Chestwrap?

chestwrap

**Comment by chaosvolt** [comment]
Tempting. Sarashi FTW? o3o

**Comment by Rivet-the-Zombie** [comment]
>Flase. Ahahah. Fuck me.

It's an easy enough mistake to make.

>Tempting. Sarashi FTW? o3o

I know it's silly for me to worry much about my character's modesty (since unsecured boobs aren't the liability in DDA that they are in real life, it's really only about modesty) in an ASCII game, but old habits die hard I guess.

**Comment by chaosvolt** [comment]
Now to hope I got it right at last. And if chest wraps already existed as an item I'd add a wool variant, but for now wanted to mainline existing content worth mainlining. Assuming 5th time's the charm...

**Comment by DavidKeaton** [comment]
I only play fem chars, so a chestwrap would be lovely. Covering your chest could be more of a boon, if the female in question is endowed, @Rivet-the-Zombie. :-)

**Comment by chaosvolt** [comment]
All is well. At laaaaast. o3o

And it would be both consistent with the other basic cloth wrap items, and good if we get a samurai profession. Maybe.

File: data/json/items/comestibles.json [hunk 3]
```diff
@@ -8655,6 +8655,67 @@
...
+        "volume" : 1,
+        "cutting" : 0,
+        "phase" : "solid",
+        "charges" : 1,
+        "bashing" : 0,
+        "fun" : -18,
+        "fun_comment" : "Who in their right mind would eat an entire garlic bulb raw?"
+    },
+    {
+        "type" : "COMESTIBLE",
```
Comment (Rivet-the-Zombie) [inline data/json/items/comestibles.json]: The game chokes on this at startup, saying it needs an ````"addiction_type" : "none",```` entry.

**Comment by chaosvolt** [comment]
Ack. Huh, odd...

**Comment by Rivet-the-Zombie** [comment]
And now it's complaining that item id garlic_seed is unknown.

Please test that your PR is working properly once you address this.

**Comment by chaosvolt** [comment]
Blargh. Right, let me see...

File: data/json/item_groups.json [hunk 4]
```diff
@@ -9692,6 +9708,8 @@
       { "item": "rhubarb", "prob": 5, "count-min": 2, "count-max": 5 },
       { "item": "wild_herbs", "prob": 10 },
       { "item": "thyme", "prob": 5, "count-min": 1, "count-max": 2 },
+      { "item": "garlic", "prob": 1 },
+      { "item": "garlic_seed", "prob": 5, "count-min": 1, "count-max": 6 },
```
Comment (Rivet-the-Zombie) [inline data/json/item_groups.json]: I've found your ````garlic_seed```` problem; it's expecting ````garlic_seed```` in item_groups, but the actual item is ````seed_garlic````.

**Comment by DavidKeaton** [comment]
id keep the `seed_garlic` naming, btw. that way it follows the `type_specific` style of naming.

**Comment by Rivet-the-Zombie** [comment]
>id keep the ````seed_garlic```` naming, btw. that way it follows the ````type_specific```` style of naming.

Agreed.

**Comment by chaosvolt** [comment]
Found it now, and got the game to stop derping after copy-pasting, so testing shit now. :v

File: data/json/items/comestibles.json [hunk 5]
```diff
@@ -8655,6 +8655,68 @@
...
+        "price" : 600,
+        "material" : "veggy",
+        "tool" : "null",
+        "volume" : 1,
+        "cutting" : 0,
+        "phase" : "solid",
+        "charges" : 1,
+        "bashing" : 0,
+        "fun" : -18,
+        "fun_comment" : "Who in their right mind would eat an entire garlic bulb raw?"
```
Comment (Rivet-the-Zombie) [inline data/json/items/comestibles.json]: I've been known to do so.

**Comment by Rivet-the-Zombie** [comment]
Looks like everything is working so far. Let me do a bit more testing, and I think you'll be good for merge.

File: data/json/items/comestibles.json (hunk 5 above)
Comment (DavidKeaton) [inline data/json/items/comestibles.json]: more of a man than I

**Comment by chaosvolt** [comment]
At laaaast. Again. o3o

**Comment by chaosvolt** [comment]
Meep. And thank you for the merge. ^^

**Pull request merged**